scripts/schema.sql

```diff
@@ -0,0 +1,14 @@
+
+CREATE MATERIALIZED VIEW IF NOT EXISTS processed.classifications AS (
```

Comment:
Any reason why these views are created in the "processed" namespace? The way I understood, the "processed" namespace was for segregating data that is not used by the telemetry_backend app but by the telemetry_processing app. But since these materialized views are used by the telemetry_backend app, they need not be within the "processed" namespace.